packages/nextjs-contentful-renderer/src/getPageContent/getPageContent.ts

```diff
@@ -19,18 +20,21 @@ export const getPageContent = async (
 
   const existingPageId = getVisitedPageIdFromCookies(context, slug);
   if (existingPageId) {
-    const page = await getPageById(apolloClient, existingPageId, preview);
+    const { modelData, pageId } = existingPageId;
+    const page = await getPageById(apolloClient, pageId, preview, modelData);
     if (page) {
       return page;
     }
   }
 
-  const route = await getRouteBySlug(apolloClient, slug, preview, domain);
-  if (!route) return undefined;
+  const replicaRoute = await getRouteBySlug(apolloClient, slug, preview, domain);
```

Comment:
any reason to rename this?

Reply:
I was just making it more relevant for the response that is returned, since it should be either a `replica` or a `route`.

Reply:
👍